internal/daysteps/daysteps.go

```diff
@@ -1,13 +1,65 @@
 package daysteps
 
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Yandex-Practicum/tracker/internal/personaldata"
+	"github.com/Yandex-Practicum/tracker/internal/spentenergy"
+)
+
 type DaySteps struct {
 	// TODO: добавить поля
+	Steps    int
+	Duration time.Duration
+	personaldata.Personal
 }
 
 func (ds *DaySteps) Parse(datastring string) (err error) {
 	// TODO: реализовать функцию
+	parts := strings.Split(datastring, ",")
+
+	if len(parts) != 2 {
+		return errors.New("длина слайса не равна 2")
+	}
+
+	steps, err := strconv.Atoi(parts[0])
+	if err != nil {
+		return errors.New("ошибка преобразования количества шагов")
```

Comment:
1. Теряешь сообщение об ошибки
2. Ошибку лучше писать на английском
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```

Reply:
Иван, спасибо большое за обратную связь !

Reply:
Иван, привет !